src/main/java/games/strategy/util/Version.java

```diff
@@ -131,42 +135,41 @@ public int hashCode() {
   }
 
   @Override
-  public int compareTo(final Version o) {
-    return o == null ? -1 : compareTo(o, false);
+  public int compareTo(final Version other) {
```

Comment:
Are the only calls to this method from within this class?
Or are all other calls comparing with 0?
In the latter case we should probably just call equals instead.

Reply:
There aren't any direct calls to `compareTo(Version)`.  I thought there may be some indirect calls from the `Comparator` implementations in the lobby client UI, but the "version" columns displayed in the lobby are actually strings, not `Version`s.  I set a breakpoint in `compareTo(Version)` and connected to the lobby; the breakpoint was never hit from the UI code.  That may imply that implementing `Comparable` could be removed.

However, the `isGreaterThan()` and `isLessThan()` methods are still required.  Their presence indicates that `Version` _should_ implement `Comparable`.  Since `Comparable` is the canonical way to determine if one instance is less/greater than another, I'm wondering if we should actually remove `isGreaterThan()` and `isLessThan()` and replace them with calls to `compareTo()`?  The obvious problem with that at this time is the additional overload `isGreaterThan(Version, boolean)`.